scripts/build-types.mjs

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-console, no-process-exit, functional/no-expression-statement */
```

Comment:
We can disable these rules in [`.eslintrc.js`](https://github.com/johnhooks/dinero.js/blob/6b1abb2854aa1845ed9ef61c392c0feb256d9816/.eslintrc.js) using the `overrides` option with `files: ['scripts/**/*']`.